example/lib/main.dart

```diff
@@ -23,11 +22,10 @@ class MyGame extends BaseGame {
 
   @override
   Future<void> onLoad() async {
-    final size = Vector2.all(100);
     svgInstance = await loadSvg('android.svg');
-    android = SvgComponent.fromSvg(size, svgInstance);
-    android.x = 100;
-    android.y = 100;
+    final size = Vector2.all(100);
+    final android = SvgComponent.fromSvg(size, svgInstance)
```

Comment:
we should probably update it so you can provide position on the constructor?

Reply:
Done!